src/lib389/lib389/cli_idm/account.py

```diff
@@ -118,6 +119,48 @@ def subtree_status(inst, basedn, log, args):
         _print_entry_status(status, entry.dn, log)
 
 
+def bulk_update(inst, basedn, log, args):
+    basedn = _get_dn_arg(args.basedn, msg="Enter basedn to search")
+    search_filter = "(objectclass=*)"
+    scope = ldap.SCOPE_SUBTREE
+    if args.scope == "one":
+        scope = ldap.SCOPE_ONELEVEL
+    if args.filter:
+        search_filter = args.filter
+    log.info(f"Searching '{basedn}' filter '{search_filter}' scope '{scope}' ...")
+    entry_list = Accounts(inst, basedn).filter(search_filter, scope=scope)
+    if not entry_list:
+        raise ValueError(f"No entries were found.")
+    log.info(f"Found {len(entry_list)} matching entries.")
+
+    failed_list = []
+    success_list = []
+    for entry in entry_list:
+        if entry.dn.lower() == basedn.lower():
+            # skip parent
+            failed_list.append(entry.dn + " (Base DN Entry Skipped)")
+            continue
+        try:
+            _generic_modify_dn(inst, basedn, log.getChild('_generic_modify_dn'), MANY, entry.dn, args)
+            success_list.append(entry.dn)
+        except ldap.LDAPError as e:
+            if "desc" in e.args[0]:
+                failed_list.append(entry.dn + f" ({e.args[0]['desc']})")
+            else:
+                failed_list.append(entry.dn + f" ({str(e)})")
```

Comment:
Could you please also report the `Failed to update` error here to `log.debug`? I think it'll make it easier to debug the operation when `-v` is used.